### PR TITLE
keys/starkkey: return StarkKeyStr as public key

### PR DESCRIPTION
**Pull request by cfal**
BCI-353

- don't use the account contract address (calculated from the hardcoded contract hash and salt) as the public key on export. note that the field isn't used on import (`FromEncryptedJSON`)
- remove AccountAddr from starknet key web presenter

### CODE REVIEW

File: core/cmd/starknet_keys_commands.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ type StarkNetKeyPresenter struct {
 
 // RenderTable implements TableRenderer
 func (p StarkNetKeyPresenter) RenderTable(rt RendererTable) error {
-	headers := []string{"ID", "Account Address", "Stark Public Key"}
+	headers := []string{"ID", "Stark Public Key"}
```
Comment (jmank88) [inline core/cmd/starknet_keys_commands.go:16]: nit/ Is it important to say `Stark` here?
```suggestion
	headers := []string{"ID", "Public Key"}
```

**Review by jmank88 — APPROVED** [review]

File: core/cmd/starknet_keys_commands.go (hunk 1 above)
Comment (jmank88) [reply core/cmd/starknet_keys_commands.go:16]: nbd :shrug:

**Review by archseer — APPROVED** [review]
Ditto here, LGTM but I'd like @aalu1418 to confirm the impact

**Review by aalu1418 — APPROVED** [review]
this PR lgtm!

basically just removing the logic for returning the account address on the API/CLI

**Pull request merged**